src/test-util.ts

```diff
@@ -72,21 +71,23 @@ type LogResponse = {
   data: unknown[];
 };
 
-export async function initializeNewTab(event: MessageEvent): Promise<void> {
-  const {tabId, path} = event.data;
-  const {opener} = window;
-  if (opener === null) {
-    return;
-  }
+export async function initializeNewTab(
+  tabId: string,
+  path: string,
+  event: StorageEvent,
+): Promise<void> {
+  const send = (message: unknown) => {
```

Comment:
We have been using `JSONValue` for these but `unknown` is OK too.

Reply:
Done.